concepts/05 Widgets/Scheduler/050 Timetable.md

```diff
@@ -209,6 +292,100 @@ For a more detailed customization, define custom templates for cells, time scale
 
     ko.applyBindings(viewModel);
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxScheduler
+            :data-source="schedulerData"
+            :current-date="currentDate"
+            :show-all-day-panel="false"
+            data-cell-template="dataCellTemplate"
+            date-cell-template="dateCellTemplate"
+            time-cell-template="timeCellTemplate"
+            current-view="week"
+        >
+            <template #dataCellTemplate="data">
+                <div v-bind:style="styles"></div>
+            </template>
+            <template #dateCellTemplate="{ data }">
+                <b v-bind:style="{ color: 'green' }">{{data.text}}</b>     
+            </template>
+            <template #timeCellTemplate="{ data }">
+                <b v-bind:style="{ color: 'green' }">{{data.text}}</b>     
+            </template>
+        </DxScheduler>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxScheduler } from 'devextreme-vue/scheduler';
+
+    export default {
+        components: {
+            DxScheduler
+        },
+        data() {
+            return {
+                currentDate: new Date(2016, 4, 24),
+                styles: {
+                    'background-color': 'rgba(86, 202, 133, 0.1)',
+                    width: '100%',
+                    height: '40px'
+                }
+            };
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { Scheduler } from 'devextreme-react/scheduler';
+
+    const currentDate = new Date(2016, 4, 24);
+
+    class App extends React.Component {
+        render() {
+            return (
+                <Scheduler
+                    dataSource={schedulerData}
+                    defaultCurrentDate={currentDate}
+                    showAllDayPanel={false}
+                    dataCellTemplate={this.dataCellTemplate}
+                    dateCellTemplate={this.dateCellTemplate}
+                    timeCellTemplate={this.timeCellTemplate}
+                    defaultCurrentView="week"
+                />
+            );
+        }
+
+        dataCellTemplate(data, index, element) {
+            return $('<div />')
```

Comment:
You are using jQuery to declare templates in React. Please refer to the [Markup Customization](https://js.devexpress.com/Documentation/Guide/React_Components/Component_Configuration_Syntax/#Markup_Customization) article for information on how to properly customize widget elements in React.